src/command.rs

```diff
@@ -55,14 +55,14 @@ pub fn get_command_runner() -> fn(
     cmd: &str,
     dir: Option<&str>,
     stream_io: bool,
-    env_vars: Option<HashMap<String, String>>,
+    env_vars: Option<&HashMap<String, String>>,
```

Comment:
Arguably the keys and values could be borrowed as well for the immediate use case of rusty-hook consuming this to support making the git params available as an env variable to the invoked scripts. However, more generally it's better for these to be owned to avoid any weird lifetime requirements on the key/values vs. the map itself.